connectors/src/connectors/slack/lib/slack_client.ts

```diff
@@ -52,6 +52,22 @@ export async function getSlackClient(
         // @ts-expect-error can't get typescript to be happy with this, but it works.
         return await Reflect.apply(target, thisArg, argumentsList);
       } catch (e) {
+        // If we get rate limited, we throw a known error. The activity will be
+        // automatically retried, no need to generate an alert
+        // Note: a previous version using slackError.code === ErrorCode.RateLimitedError failed
+        // see PR #2689 for details
+        if (
+          e instanceof Error &&
+          e.message.startsWith("A rate limit was exceeded")
+        ) {
+          throw {
+            __is_dust_error: true,
+            message: e.message,
+            triggeringError: e,
+            type: "connector_rate_limit_activity_error",
```

Comment:
I would not mention activity in the error type here.
Can we just replace the type `connector_rate_limit_activity_error` with `connector_rate_limit_error`

Reply:
Still mentioning activity here.